src/search/models.rs

```diff
@@ -0,0 +1,82 @@
+use meilisearch_sdk::document::Document;
+use serde::{Serialize, Deserialize};
+//TODO: Move theses structs to the needed place, and convert everything to a standard mod.
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct Attachment {
+    pub url: String,
+    pub thumbnail_url: String,
+    pub is_default: bool,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Category {
+    pub name: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Author {
+    pub name: String,
+    pub url: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct CurseVersion {
+    pub game_version: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct CurseForgeMod {
+    pub id: i32,
+    pub name: String,
+    pub authors: Vec<Author>,
+    pub attachments: Vec<Attachment>,
+    pub website_url: String,
+    pub summary: String,
+    pub download_count: f32,
+    pub categories: Vec<Category>,
+    pub game_version_latest_files: Vec<CurseVersion>,
+    pub date_created: String,
+    pub date_modified: String,
+    pub game_slug: String,
+}
```

Comment:
Might want to move this to the curseforge indexer, as this is curseforge code

Reply:
Yup, everything was just separated to be more clear